shared/settings/files/index.tsx

```diff
@@ -202,7 +203,11 @@ const styles = Styles.styleSheetCreate({
     alignItems: 'center',
   },
   syncNotificationDropdownItem: {
-    width: 2 * Styles.globalMargins.xlarge,
+    alignItems: 'center',
+    paddingLeft: Styles.globalMargins.small,
+  },
+  syncNotificationDropdownOverlay: {
+    width: Styles.globalMargins.xlarge + Styles.globalMargins.medium,
```

Comment:
Why is this expressed as a sum of margins instead of as a number?

Reply:
I always try to express dimensions as linear combinations of margins so it looks "good" relative to all the other dimensions we use, but usually I change it to an actual amount after the fact. I can do that here.